submissions/Semka094/dom/script.js

```diff
@@ -0,0 +1,123 @@
+        items: [
+          'Males medium to large 4.5 to 8.1 kg; Females 2.7 to 5.4 kg. Most Bengals have green, yellow or gold eyes. ' +
+          'The lynx points have blue eyes, and the minks have aqua eyes.Bengals are long and lean and athletic in ' +
+          'appearance. Bengals are larger than the average house cat because of their muscular bodies. Bengal kittens ' +
+          'range between $3,000 and $5,000. Adult Bengal cats range between $700 and $1500.'
+        ]
+      }
+    ]
+  },
+  health: {
+    sections: [
+      {
+        title: 'Health ',
+        items: [
+          'The typical Bengal cat lifespan is 9–15 years, and they\'re a relatively healthy breed. However, as with all' +
+          ' cats, they do have a few health issues you need to keep in mind before you purchase a new pet. Most serious ' +
+          'breeders are careful to breed animals without genetic health problems, but three of the most common maladies' +
+          ' among Bengals are:'
+        ]
+      },
+      {
+        title: 'Heart disease:',
+        items: [
+          'Heart disease in Bengal cats is called hypertrophic cardiomyopathy, which can cause the heart muscle to ' +
+          'thicken, particularly in older animals. This can result in blood clots or congestive heart failure and a ' +
+          'shorter lifespan. '
+        ]
+      },
+      {
+        title: 'Eye disease:',
+        items: [
+          'Bengals can also get an eye condition called progressive retinal atrophy, which can cause deterioration of ' +
+          'the retina and eventual blindness.'
+        ]
+      },
+      {
+        title: 'Anesthetic allergies:',
+        items: [
+          'Bengals can be extremely sensitive to anesthetics and should be watched carefully during any surgeries—including ' +
+          'neutering and spaying. An allergic reaction to anesthetics can result in cardiac arrest.'
+        ]
+      }
+    ]
+    
+  },
+  facts: {
+    sections: [
+      {
+        title: 'Chatty and vocal',
+        items: [
+          "Bengal cats are also known to be a bit chatty with their owners. They won't meow excessively, but they're " +
+          "pros at telling you exactly what they need, especially if it's \"my food bowl is almost empty,\" or \"I want " +
+          "to play.\" A Bengal also won't sit idly by after you come home from a long day at work. You can expect a " +
+          "royal greeting, complete with a serenade."
+        ],
+      },
+    ]
+  },
+}
+
+document.getElementById('menu').addEventListener('click', (e) => {
+  const menuId = e.target.dataset.id;
+  const menuEntity = data[menuId];
+
+  const pageSectionsHtml = menuEntity.sections.map(({title, items}) => {
+    const titleHtml = title ? `<h3>${title}</h3>` : '';
+    const textHtml = items.map((text) => `<p>${text}</p>`).join('');
+
+    return `${titleHtml}${textHtml}`
+  });
+
+
+  const contentElement = document.getElementById('content');
```

Comment:
You better use the class name to search.

Reply:
I think it is better to use ID since it is unique and the class names might be used in other parts of the project. 
In this case, I will be forced to rely on the order of elements in the DOM structure.
What do you think?